src/thread_timer.h

```diff
@@ -75,9 +83,11 @@ class ThreadTimer {
 
   // Accumulated time so far (does not contain current slice if running_)
   double real_time_used_ = 0;
+
   double cpu_time_used_ = 0;
   // Manually set iteration time. User sets this with SetIterationTime(seconds).
   double manual_time_used_ = 0;
+  double manual_time_used2_ = 0;
```

Comment:
So we need to track every timer as-is *and* as as a sum-of-squares?

Reply:
Yes, the idea is to look at the time durations as independent identically distributed samples of a random variable. And then use second-order statistics to estimate mean, variance, and accuracy of the estimate of the mean. So for this, we need indeed to keep track of the sum of the squares of the time durations.